services/validatorWorker/lib/sentryInterface.js

```diff
@@ -38,9 +39,13 @@ function SentryInterface(adapter, channel, opts = { logging: true }) {
 
 	this.getLatestMsg = function(from, type) {
 		const url = `${baseUrl}/validator-messages/${from}/${type}?limit=1`
-		return fetchJson(url).then(({ validatorMessages }) =>
-			validatorMessages.length ? validatorMessages[0].msg : null
-		)
+		return fetchJson(url).then(({ validatorMessages }) => {
+			if (validatorMessages.length) {
+				const { err } = sentry.message.validate(validatorMessages)
+				return !err ? validatorMessages[0].msg : null
```

Comment:
silent errors are not a good idea; 

perhaps throw here and see what happens? I think it will be caught by the tick `catch` and the tick would be aborted, which is OK